pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -22,6 +26,7 @@ type SecurityPolicyEnforcer interface {
 	EnforceOverlayMountPolicy(containerID string, layerPaths []string) (err error)
```

Comment:
I am curious if long-term (not in this PR) the `SecurityPolicyEnforcer` code should be separated out from `pkg/securitypolicy` and moved somewhere like under `internal/guest`. Then `pkg/securitypolicy` could just be the "outward facing" definitions needed to construct a security policy, and all the implementation specific logic to enforce a policy would be close to the rest of the GCS code.

Thoughts?

Reply:
For instance it would be nice if nothing in `pkg/securitypolicy` had to take a dependency on stuff under `internal`.

Reply:
yeah, I had similar thoughts, that's one of the reasons I why put linux labels and broke some dependencies.

Reply:
Note that everything was moved into this package as part of earlier reviews. I believe you were one of the people who requested that @kevpar although I can't remember the reasoning. I bring this up not because I am opposed to such a change but to perhaps jog your memory as to why you asked for them to be in a single package. That reason might not be relevant anymore, I can't remember afterall.

Reply:
I'll confess to not remembering the full context back then either. It's entirely possible that I have changed my mind after getting a fuller sense of the code structure. :) My current feeling looking at the code is that the enforcement piece is tightly coupled to the GCS and bridge specifically, in that it is effectively a state machine operating based on bridge requests, and using them to build up its context for decision making. In that light, it seems reasonable to me that it be moved closer to the GCS code.

Reply:
I think your description as a state machine operating based on bridge requests is spot on @kevpar. When Maksim does the protocol enforcement that will be fully brought to the forefront.

Reply:
agree as well.